### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by migueltarga**

### CODE REVIEW

**Review by Divala — COMMENTED** [review]
The app is crashing and am getting that error in logcat:
models.Post.getThumb_url()' on a null object reference

File: android/app/src/main/java/com/google/firebase/samples/apps/friendlypix/FeedsActivity.java [hunk 1]
```diff
@@ -109,10 +115,14 @@ public void onCancelled(DatabaseError firebaseError) {
         });
     }
 
+
```
Comment (Divala) [inline android/app/src/main/java/com/google/firebase/samples/apps/friendlypix/FeedsActivity.java:118]: whats new here?

Comment (migueltarga) [reply android/app/src/main/java/com/google/firebase/samples/apps/friendlypix/FeedsActivity.java:118]: nothing, extra line break, by mistake.

**Comment by Divala** [comment]
Okay thanks but it's crashing on taking a followed post getThumb_url () is
throwing null object reference

**Comment by migueltarga** [comment]
I will take a look

File: android/app/src/main/java/com/google/firebase/samples/apps/friendlypix/FeedsActivity.java (hunk 1 above)
Comment (Divala) [reply android/app/src/main/java/com/google/firebase/samples/apps/friendlypix/FeedsActivity.java:118]: Okay

**Comment by Divala** [comment]
I tried to debug but may be there is a conflict when loading home and feed tabs. whats your finding?

**Comment by migueltarga** [comment]
I didn't have time to look into this yet, if you find anything, let me know, thanks!

**Pull request closed without merging**